src/linter/root.go

```diff
@@ -696,6 +696,15 @@ func (d *RootWalker) lowerCaseModifier(m *node.Identifier) string {
 }
 
 func (d *RootWalker) enterPropertyList(pl *stmt.PropertyList) bool {
+	sc := meta.NewScope()
+	sc.SetInClosure(false)
```

Comment:
I'm pretty sure a Scope is "out of closure" by default, so this call is redundant.